vivisect/extensions/__init__.py

```diff
@@ -14,33 +15,83 @@
 (vw) and an vivisect gui reference (if present).
 '''
 
-def loadExtensions( vw, vwgui ):
+logger = logging.getLogger(__name__)
 
+
+def earlyExtensions(vw):
+    logger.debug('earlyLoad')
+    for mod in extensions:
+        if not hasattr(mod, 'earlyLoad'):
+            logger.debug('earlyLoad: %r (no earlyLoad() function)', mod)
+            continue
+
+        logger.debug('earlyLoad: %r', mod)
+        try:
+            mod.earlyLoad(vw)
+        except Exception, e:
```

Comment:
i think this is py2.5 syntax for exceptions
```suggestion
        except Exception as e:
```

Reply:
wow.  i guess i haven't looked at this in a while.  this should be labeled "WIP" for conversation...
thanks for pointing it out, @williballenthin !